src/app/models/base.py

```diff
@@ -70,6 +70,63 @@ class ColumnOutput(BaseModel):
         default=0, description=PERCENTAGE_DESCRIPTOR
     )
     zum_spider: Optional[float] = Field(default=0, description=PERCENTAGE_DESCRIPTOR)
+    leifi_spider: Optional[float] = Field(default=0, description=PERCENTAGE_DESCRIPTOR)
```

Comment:
Mid term I think it would be great if we can eliminate the hard coded sources here and turn this into something configurable at runtime - or even request time (e.g. the request could list the set of relevant spiders). Otherwise, the code needs to be changed whenever a new spider is added or removed.

Reply:
Yeah, I guess we talked about this today. But to unblock I suppose we change that in a follow up PR!